src/cascadia/TerminalSettingsEditor/Appearances.h

```diff
@@ -31,30 +31,27 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     struct Font : FontT<Font>
     {
     public:
-        Font(winrt::hstring name, winrt::hstring localizedName, IDWriteFontFamily* family) :
+        Font(winrt::hstring name, winrt::hstring localizedName, wil::com_ptr<IDWriteFontFamily> family) :
```

Comment:
technically considered rude to force a caller into a specific container type, but.. that's OK

Reply:
I think in this case it's perfectly fine, if not optimal. If the caller already has this particular container type, then they can `std::move` it. If they don't they can choose how to construct it. It's similar to taking constructor parameters by value as `std::string` and moving it into a member VS taking a `string_view` and allocating a new `string` member every time, right?

Reply:
totally fair